build.ps1

```diff
@@ -33,4 +33,4 @@ if (!(Test-Path $buildFolder)) {
     }
 }
 
-&"$buildFile" $args
\ No newline at end of file
+&"$buildFile" @args
```

Comment:
Question: Do we require PowerShell 3.0 somewhere? Splatting isn't available earlier.